### ISSUE TITLE
Not rounding after discount code has been applied on Magento V. 2.3.1 | PHP 7.3 for Magento 2.3.4 not supported

### ISSUE THREAD

**Issue by w7tf** [comsolit/mage2_rappenrunden#2]
First of all thank you very much for providing such a plugin for free.

Issue description on Magento 2.3.1:

Installed extension manually.
Enabled it in the backend
Applied coupon code
Order Value is not being rounded in cart as well as on checkout

Issue description on Magento 2.3.4:

When trying to install via composer following error is recieved:
Package comsolit/rappenrunden at version  has a PHP requirement incompatible with your PHP version (7.3.15)

I would want to contribute to this extention but unfortionatley my knowledge with Magento extentions and coding is way to little.

**Comment by tecjam** [comment]
Hi @warum7 

I've pushed version 0.0.3 to the dev branch. Could you please test the dev branch using:

`composer require comsolit/rappenrunden:dev-dev`

Fixes in 0.0.3:
```
- New Feature: allow switching between 0.05 and 0.1 rounding
- Dropped PHP 5.5 support;
- Added PHP 7.3 and 7.4 support
- Fixed issues with tier prices
```

**Comment by w7tf** [comment]
> Hi @warum7
> 
> I've pushed version 0.0.3 to the dev branch. Could you please test the dev branch using:
> 
> `composer require comsolit/rappenrunden:dev-dev`
> 
> Fixes in 0.0.3:
> 
> ```
> - New Feature: allow switching between 0.05 and 0.1 rounding
> - Dropped PHP 5.5 support;
> - Added PHP 7.3 and 7.4 support
> - Fixed issues with tier prices
> ```

Hey @tecjam 

Thank you very much for letting me test the new version! 

Tested both features (0.05 & 0.1) both seem to be working without any issues. Please note that I am not using Paypal/Braintree as checkout options. The reason I state that is that those modules sometimes cause issues.

Anyways I have tested it with the Payrexx Plugin and the Offline Payment methods and it works.

Thank you very much for your efforts, appreciated!
